From 8a44067838e268db88ea2158258f8006cb1344b7 Mon Sep 17 00:00:00 2001
From: Andrew Gregory <andrew.gregory.8@gmail.com>
Date: Tue, 12 Feb 2013 17:09:50 -0500
Subject: [PATCH] Fix shebangs in services to point to the correct location of
 runscript

SBINDIR and BINDIR can be set independently of PREFIX.  This fixes
broken shebangs in service files when SBINDIR is set to something other
than PREFIX/sbin

Signed-off-by: Andrew Gregory <andrew.gregory.8@gmail.com>
---
 init.d.misc/avahi-dnsconfd.in | 2 +-
 init.d.misc/avahid.in         | 2 +-
 init.d.misc/dbus.in           | 2 +-
 init.d.misc/dhcpcd.in         | 2 +-
 init.d.misc/dnsmasq.in        | 2 +-
 init.d.misc/hald.in           | 2 +-
 init.d.misc/named.in          | 2 +-
 init.d.misc/ntpd.in           | 2 +-
 init.d.misc/openvpn.in        | 2 +-
 init.d.misc/polkitd.in        | 2 +-
 init.d.misc/sshd.in           | 2 +-
 init.d.misc/wpa_supplicant.in | 2 +-
 init.d/adjkerntz.in           | 2 +-
 init.d/bootmisc.in            | 2 +-
 init.d/consolefont.in         | 2 +-
 init.d/devd.in                | 2 +-
 init.d/devdb.in               | 2 +-
 init.d/devfs.in               | 2 +-
 init.d/dmesg.in               | 2 +-
 init.d/dumpon.in              | 2 +-
 init.d/encswap.in             | 2 +-
 init.d/fsck.in                | 2 +-
 init.d/hostid.in              | 2 +-
 init.d/hostname.in            | 2 +-
 init.d/hwclock.in             | 2 +-
 init.d/ipfw.in                | 2 +-
 init.d/keymaps.in             | 2 +-
 init.d/killprocs.in           | 2 +-
 init.d/local.in               | 2 +-
 init.d/localmount.in          | 2 +-
 init.d/mixer.in               | 2 +-
 init.d/modules.in             | 2 +-
 init.d/mount-ro.in            | 2 +-
 init.d/moused.in              | 2 +-
 init.d/mtab.in                | 2 +-
 init.d/net.lo.in              | 2 +-
 init.d/netmount.in            | 2 +-
 init.d/network.in             | 2 +-
 init.d/newsyslog.in           | 2 +-
 init.d/nscd.in                | 2 +-
 init.d/numlock.in             | 2 +-
 init.d/pf.in                  | 2 +-
 init.d/powerd.in              | 2 +-
 init.d/procfs.in              | 2 +-
 init.d/rarpd.in               | 2 +-
 init.d/rc-enabled.in          | 2 +-
 init.d/root.in                | 2 +-
 init.d/rpcbind.in             | 2 +-
 init.d/savecache.in           | 2 +-
 init.d/savecore.in            | 2 +-
 init.d/staticroute.in         | 2 +-
 init.d/swap-blk.in            | 2 +-
 init.d/swap.in                | 2 +-
 init.d/swapfiles.in           | 2 +-
 init.d/swclock.in             | 4 ++--
 init.d/syscons.in             | 2 +-
 init.d/sysctl.BSD.in          | 2 +-
 init.d/sysctl.Linux.in        | 2 +-
 init.d/sysfs.in               | 2 +-
 init.d/syslogd.in             | 2 +-
 init.d/termencoding.in        | 2 +-
 init.d/tmpfiles.setup.in      | 2 +-
 init.d/ttys.in                | 2 +-
 init.d/urandom.in             | 2 +-
 init.d/wscons.in              | 2 +-
 sh/gendepends.sh.in           | 4 ++--
 66 files changed, 68 insertions(+), 68 deletions(-)

diff --git a/init.d.misc/avahi-dnsconfd.in b/init.d.misc/avahi-dnsconfd.in
index 423ec91..abe2d22 100644
--- a/init.d.misc/avahi-dnsconfd.in
+++ b/init.d.misc/avahi-dnsconfd.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/avahid.in b/init.d.misc/avahid.in
index ebda61c..ea2241b 100644
--- a/init.d.misc/avahid.in
+++ b/init.d.misc/avahid.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/dbus.in b/init.d.misc/dbus.in
index 94057b9..d0004e7 100644
--- a/init.d.misc/dbus.in
+++ b/init.d.misc/dbus.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/dhcpcd.in b/init.d.misc/dhcpcd.in
index 9e8469e..b22d4d1 100644
--- a/init.d.misc/dhcpcd.in
+++ b/init.d.misc/dhcpcd.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/dnsmasq.in b/init.d.misc/dnsmasq.in
index 1d5c9aa..bdb0411 100644
--- a/init.d.misc/dnsmasq.in
+++ b/init.d.misc/dnsmasq.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/hald.in b/init.d.misc/hald.in
index 7c28760..741442f 100644
--- a/init.d.misc/hald.in
+++ b/init.d.misc/hald.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/named.in b/init.d.misc/named.in
index 4dafd88..924f5b5 100644
--- a/init.d.misc/named.in
+++ b/init.d.misc/named.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/ntpd.in b/init.d.misc/ntpd.in
index 0036feb..6f18521 100644
--- a/init.d.misc/ntpd.in
+++ b/init.d.misc/ntpd.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/openvpn.in b/init.d.misc/openvpn.in
index ef37e19..7915623 100644
--- a/init.d.misc/openvpn.in
+++ b/init.d.misc/openvpn.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/polkitd.in b/init.d.misc/polkitd.in
index 2fac385..fda0764 100644
--- a/init.d.misc/polkitd.in
+++ b/init.d.misc/polkitd.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/sshd.in b/init.d.misc/sshd.in
index 20be770..7efb666 100644
--- a/init.d.misc/sshd.in
+++ b/init.d.misc/sshd.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d.misc/wpa_supplicant.in b/init.d.misc/wpa_supplicant.in
index 22d7e0f..f908e36 100644
--- a/init.d.misc/wpa_supplicant.in
+++ b/init.d.misc/wpa_supplicant.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/adjkerntz.in b/init.d/adjkerntz.in
index 76ad175..d1349c8 100644
--- a/init.d/adjkerntz.in
+++ b/init.d/adjkerntz.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/bootmisc.in b/init.d/bootmisc.in
index 4fdb433..565f6f2 100644
--- a/init.d/bootmisc.in
+++ b/init.d/bootmisc.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/consolefont.in b/init.d/consolefont.in
index 0e91d13..61e76c3 100644
--- a/init.d/consolefont.in
+++ b/init.d/consolefont.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/devd.in b/init.d/devd.in
index b474b75..395dfa7 100644
--- a/init.d/devd.in
+++ b/init.d/devd.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/devdb.in b/init.d/devdb.in
index fcf2597..9213758 100644
--- a/init.d/devdb.in
+++ b/init.d/devdb.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/devfs.in b/init.d/devfs.in
index 2834654..36955f3 100644
--- a/init.d/devfs.in
+++ b/init.d/devfs.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/dmesg.in b/init.d/dmesg.in
index 1188483..1c72f76 100644
--- a/init.d/dmesg.in
+++ b/init.d/dmesg.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/dumpon.in b/init.d/dumpon.in
index 4450bf9..fdbdd49 100644
--- a/init.d/dumpon.in
+++ b/init.d/dumpon.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/encswap.in b/init.d/encswap.in
index bb52735..57cc762 100644
--- a/init.d/encswap.in
+++ b/init.d/encswap.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright 1992-2012 FreeBSD Project
 # Released under the 2-clause BSD license
 
diff --git a/init.d/fsck.in b/init.d/fsck.in
index 8322452..8c460f7 100644
--- a/init.d/fsck.in
+++ b/init.d/fsck.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/hostid.in b/init.d/hostid.in
index b682b3d..e98ff91 100644
--- a/init.d/hostid.in
+++ b/init.d/hostid.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/hostname.in b/init.d/hostname.in
index 5eea7f0..4374548 100644
--- a/init.d/hostname.in
+++ b/init.d/hostname.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/hwclock.in b/init.d/hwclock.in
index eb44f62..0f94993 100644
--- a/init.d/hwclock.in
+++ b/init.d/hwclock.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/ipfw.in b/init.d/ipfw.in
index 434920d..a455e29 100644
--- a/init.d/ipfw.in
+++ b/init.d/ipfw.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/keymaps.in b/init.d/keymaps.in
index 5889268..7abd731 100644
--- a/init.d/keymaps.in
+++ b/init.d/keymaps.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/killprocs.in b/init.d/killprocs.in
index 7f1cf91..5c95925 100644
--- a/init.d/killprocs.in
+++ b/init.d/killprocs.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/local.in b/init.d/local.in
index 87cca7d..fb2f9b1 100644
--- a/init.d/local.in
+++ b/init.d/local.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/localmount.in b/init.d/localmount.in
index 7d6c305..9cbbd41 100644
--- a/init.d/localmount.in
+++ b/init.d/localmount.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/mixer.in b/init.d/mixer.in
index a08aea1..a658e57 100644
--- a/init.d/mixer.in
+++ b/init.d/mixer.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/modules.in b/init.d/modules.in
index 72f14e7..0d6d21b 100644
--- a/init.d/modules.in
+++ b/init.d/modules.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/mount-ro.in b/init.d/mount-ro.in
index 41b0b9e..f8d7258 100644
--- a/init.d/mount-ro.in
+++ b/init.d/mount-ro.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/moused.in b/init.d/moused.in
index cdcf36a..ffbed76 100644
--- a/init.d/moused.in
+++ b/init.d/moused.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/mtab.in b/init.d/mtab.in
index a32b8d6..beee3e1 100644
--- a/init.d/mtab.in
+++ b/init.d/mtab.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/net.lo.in b/init.d/net.lo.in
index b6773fe..95744f8 100644
--- a/init.d/net.lo.in
+++ b/init.d/net.lo.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/netmount.in b/init.d/netmount.in
index fd577d5..dbf921b 100644
--- a/init.d/netmount.in
+++ b/init.d/netmount.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/network.in b/init.d/network.in
index 13895f0..898b0a8 100644
--- a/init.d/network.in
+++ b/init.d/network.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/newsyslog.in b/init.d/newsyslog.in
index 6cf72e6..1cbbe95 100644
--- a/init.d/newsyslog.in
+++ b/init.d/newsyslog.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/nscd.in b/init.d/nscd.in
index 6d3c0c5..41e42ae 100644
--- a/init.d/nscd.in
+++ b/init.d/nscd.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/numlock.in b/init.d/numlock.in
index 3cb8bee..433ba93 100644
--- a/init.d/numlock.in
+++ b/init.d/numlock.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/pf.in b/init.d/pf.in
index 1097ac8..aed6539 100644
--- a/init.d/pf.in
+++ b/init.d/pf.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/powerd.in b/init.d/powerd.in
index 10ffecc..88f050d 100644
--- a/init.d/powerd.in
+++ b/init.d/powerd.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/procfs.in b/init.d/procfs.in
index 7b7c6e6..1432ecf 100644
--- a/init.d/procfs.in
+++ b/init.d/procfs.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/rarpd.in b/init.d/rarpd.in
index daacb41..8ea29ac 100644
--- a/init.d/rarpd.in
+++ b/init.d/rarpd.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/rc-enabled.in b/init.d/rc-enabled.in
index 9ba73b0..fed30ce 100644
--- a/init.d/rc-enabled.in
+++ b/init.d/rc-enabled.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/root.in b/init.d/root.in
index c8ae4f1..387f205 100644
--- a/init.d/root.in
+++ b/init.d/root.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/rpcbind.in b/init.d/rpcbind.in
index 7c70871..77d6c87 100644
--- a/init.d/rpcbind.in
+++ b/init.d/rpcbind.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/savecache.in b/init.d/savecache.in
index 581be59..3dec78b 100644
--- a/init.d/savecache.in
+++ b/init.d/savecache.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/savecore.in b/init.d/savecore.in
index c55a0ab..e256fba 100644
--- a/init.d/savecore.in
+++ b/init.d/savecore.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/staticroute.in b/init.d/staticroute.in
index 213dddf..55d0680 100644
--- a/init.d/staticroute.in
+++ b/init.d/staticroute.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/swap-blk.in b/init.d/swap-blk.in
index a7862ee..4ddf61c 100644
--- a/init.d/swap-blk.in
+++ b/init.d/swap-blk.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/swap.in b/init.d/swap.in
index fa08b4d..4d7a02c 100644
--- a/init.d/swap.in
+++ b/init.d/swap.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/swapfiles.in b/init.d/swapfiles.in
index baf7591..ec73baa 100644
--- a/init.d/swapfiles.in
+++ b/init.d/swapfiles.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/swclock.in b/init.d/swclock.in
index 077f258..465f76c 100644
--- a/init.d/swclock.in
+++ b/init.d/swclock.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
@@ -17,7 +17,7 @@ start()
 {
 	ebegin "Setting the local clock based on last shutdown time"
 	if ! swclock 2> /dev/null; then
-	swclock --warn @PREFIX@/sbin/runscript
+	swclock --warn @SBINDIR@/runscript
 	fi
 	eend $?
 }
diff --git a/init.d/syscons.in b/init.d/syscons.in
index 0666947..6a7523b 100644
--- a/init.d/syscons.in
+++ b/init.d/syscons.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/sysctl.BSD.in b/init.d/sysctl.BSD.in
index c7d2028..9f84e36 100644
--- a/init.d/sysctl.BSD.in
+++ b/init.d/sysctl.BSD.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/sysctl.Linux.in b/init.d/sysctl.Linux.in
index 119f6ee..2413154 100644
--- a/init.d/sysctl.Linux.in
+++ b/init.d/sysctl.Linux.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/sysfs.in b/init.d/sysfs.in
index 85bc083..114a653 100644
--- a/init.d/sysfs.in
+++ b/init.d/sysfs.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/syslogd.in b/init.d/syslogd.in
index 817afa1..28090f2 100644
--- a/init.d/syslogd.in
+++ b/init.d/syslogd.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/termencoding.in b/init.d/termencoding.in
index a8a96e5..d03633d 100644
--- a/init.d/termencoding.in
+++ b/init.d/termencoding.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2008-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/tmpfiles.setup.in b/init.d/tmpfiles.setup.in
index a3ecd31..81bc369 100644
--- a/init.d/tmpfiles.setup.in
+++ b/init.d/tmpfiles.setup.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright 1999-2012 Gentoo Foundation
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/ttys.in b/init.d/ttys.in
index 2adbe81..aec8340 100644
--- a/init.d/ttys.in
+++ b/init.d/ttys.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2008 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/urandom.in b/init.d/urandom.in
index 2dfe1cc..97d0407 100644
--- a/init.d/urandom.in
+++ b/init.d/urandom.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2007-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/init.d/wscons.in b/init.d/wscons.in
index 5d94eff..35bd2b0 100644
--- a/init.d/wscons.in
+++ b/init.d/wscons.in
@@ -1,4 +1,4 @@
-#!@PREFIX@/sbin/runscript
+#!@SBINDIR@/runscript
 # Copyright (c) 2008-2009 Roy Marples <roy@marples.name>
 # Released under the 2-clause BSD license.
 
diff --git a/sh/gendepends.sh.in b/sh/gendepends.sh.in
index f78f3d3..461c688 100644
--- a/sh/gendepends.sh.in
+++ b/sh/gendepends.sh.in
@@ -53,8 +53,8 @@ do
 
 		# Only generate dependencies for runscripts
 		read one two three <"$RC_SERVICE"
-		[ "$one" = "#!@PREFIX@/sbin/runscript" ] || \
-		[ "$one" = "#!" -a "$two" = "@PREFIX@/sbin/runscript" ] || \
+		[ "$one" = "#!@SBINDIR@/runscript" ] || \
+		[ "$one" = "#!" -a "$two" = "@SBINDIR@/runscript" ] || \
 			continue
 		unset one two three
 
-- 
1.8.3

